### PR TITLE
Initial build for QA

### PR DESCRIPTION
**Pull request by clpetersonucf**

### CODE REVIEW

**Comment by FrenjaminBanklin** [comment]
Wouldn't delay deploying because of it, but these inputs are a bit short considering the wide open space to the right of them.

lots-o-white-space

File: src/install.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 general:
```
Comment (iturgeon) [inline src/install.yaml]: Go ahead and make this widget visible in the catalog by default

**Comment by iturgeon** [comment]
I'm getting a `https://code.angularjs.org/1.6.9/docs/error/ngRepeat/dupes?p0=response%20in%20responses&p1=string:Sometimes&p2=Sometimes` error on the score screen

**Comment by iturgeon** [comment]
One more feedback item - I think the title is a little hard to notice.  Chances are people will miss it all together. 

Perhaps in a future phase we should do the little setup card like Labeling

**Comment by iturgeon** [comment]
Scratch that last emoji title issue - it's a Materia issue https://github.com/ucfopen/Materia/issues/909

**Comment by clpetersonucf** [comment]
Implemented the following corrections based on feedback:
- Added demo
- set `in_catalog` flag to `true`
- Expanded min-width of certain inputs
- Removed default question text and added placeholder text instead
- Fixed issue with scorescreen choking on duplicate responses in responses array
- onQuestionImportComplete should be working properly now

**Comment by iturgeon** [comment]
Importing error

`Uncaught TypeError: Cannot read property 'questionType' of undefined`


```$scope.onQuestionImportComplete = function(items) {
      var i, item, len1;
      for (i = 0, len1 = items.length; i < len1; i++) {
        item = items[i];
        $scope.cards.push({
          question: item.questions[0].text,
          questionType: item.options.questionType,
          answerType: item.options.answerType,
          answers: item.answers,
          displayStyle: item.options.displayStyle,
          group: item.options.group
        });
        questionCount++;
      }
      return $scope.$apply(function() {
        return $scope.showToast("Added " + items.length + " imported questions.");
      });
    };```

`questionType: item.options.questionType,` seems to be an issue in the mwdk anyway

**Comment by nathaned** [comment]
image

feedback on scorescreen:

- I think there should be some type of "your response has been recorded" for the survey. maybe a big banner at the top with a check
- Icons in top right were confusing when everything was MC. Once I saw the demo with the checkbox and the edit pen it made more sense, but I don't think they're really necessary

Also, in the creator, messages from `Materia.CreatorCore.cancelSave` are concatenated to "Unfortunately your progress was not saved because", so those should be reworded.

**Pull request merged**